charts/invenio/values.yaml

```diff
@@ -37,6 +37,7 @@ invenio:
         consumer_secret: ""
   extra_config: {}
   extra_env_from_secret: {}
+  uwsgi_extra_config: {}
```

Comment:
Typically, Helm values use camelCase and not snake_case. I suggest we change that.

Reply:
I have followed chart's convention 
camelCase is better.

Reply:
This chart is inconsistent with what case it uses. But do as you wish. 🙃